src/main/java/com/example/nasapicturesstealer/webclinet/HttpCallConfiguration.java

```diff
@@ -9,7 +9,6 @@
 
 @Configuration
 public class HttpCallConfiguration {
-
     private static final String PATH = "https://api.nasa.gov/mars-photos/api/v1";
```

Comment:
can be moved out to application.properties file as well as api key from  StealService